resource/Manifest-1.json

```diff
@@ -1,18 +1,19 @@
 {
   "title": "Manifest.json Schema",
   "$schema": "http://json-schema.org/draft-07/schema",
-  "$id": "https://raw.githubusercontent.com/qooxdoo/qooxdoo-compiler/master/resource/Manifest-1.0.0.json",
+  "$id": "https://raw.githubusercontent.com/qooxdoo/qooxdoo-compiler/master/resource/Manifest-1.json",
   "description": "qooxdoo's Manifest files serve to provide meta information for a library in a structured way. Their syntax is in JSON. They have a more informational part (keyed info), which is more interesting for human readers, a technical part (named provides) that is used in the processing of generator configurations, and a part named externalResources to include CSS and Javascript files.",
```

Comment:
Why not go with a full version string?

Reply:
because after sleeping over it I found that config files are more like database schemas and there it is customary to work with just an increasing number with potentially a system for upgrading the databases between releases ... but I am not deadset against semver here, just thought it might be simpler like this.

Reply:
You have a point. In the end, it is probably a question of whether the major (non-BC) and minor (BC) version change makes sense in this context. Will it trigger different things? For example, in contrib.json, it seems to make sense since there can be upgrades in information that are still BC (you can use an older file with a newer compiler) or not (you need to regenerate the file).